auto_submit/lib/revert_support/revert_info_collection.dart

```diff
@@ -0,0 +1,93 @@
+// Copyright 2024 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+class RevertInfoCollection {
+  RevertInfoCollection();
+
+  // tags that appear in the revert issue body.
+  final String startOriginalPrLinkTag = '<!-- start_original_pr_link -->';
+  final String endOriginalPrLinkTag = '<!-- end_original_pr_link -->';
+
+  final String startInitiatingAuthorTag = '<!-- start_initiating_author -->';
+  final String endInitiatingAuthorTag = '<!-- end_initiating_author -->';
+
+  final String startRevertReasonTag = '<!-- start_revert_reason -->';
+  final String endRevertReasonTag = '<!-- end_revert_reason -->';
+
+  final String startOriginalPrAuthorTag = '<!-- start_original_pr_author -->';
+  final String endOriginalPrAuthorTag = '<!-- end_original_pr_author -->';
+
+  final String startReviewersTag = '<!-- start_reviewers -->';
+  final String endReviewersTag = '<!-- end_reviewers -->';
+
+  final String startRevertBodyTag = '<!-- start_revert_body -->';
+  final String endRevertBodyTag = '<!-- end_revert_body -->';
+
+  String? extractOriginalPrLink(String text) {
+    // This one is weird and is done to match the github standard.
+    return _extract(
```

Comment:
Does it make sense to have all fields to follow same style? It would simply the logic here.

Reply:
You mean with the : separator or some other value? Yeah it does. Initially I was following the convention github uses but I guess we do not need that any longer. I will take a look at simplifying.

Reply:
Okay I am going to create a new change for this.

Reply:
See https://github.com/flutter/cocoon/pull/3530